functions/src/Controller/Elections.ts

```diff
@@ -0,0 +1,3 @@
+import { Elections, electionsConverter } from '../Models/Elections';
+import * as functions from 'firebase-functions';
+import { firestore } from 'firebase-admin';
```

Comment:
Ahhh what is this is this a firestore?????????????????? Oh God my eyes, this should be db, @Sebastian1712 change it now change it change it it hurts my eyesssss